exe/kubernetes-deploy

```diff
@@ -23,6 +26,7 @@ ARGV.options do |opts|
   opts.on("--allow-protected-ns", "Enable deploys to #{prot_ns}; requires --no-prune") { allow_protected_ns = true }
   opts.on("--no-prune", "Disable deletion of resources that do not appear in the template dir") { prune = false }
   opts.on("--template-dir=DIR", "Set the template dir (default: config/deploy/$ENVIRONMENT)") { |v| template_dir = v }
+  opts.on("--use-stdin", "Use $stdin instead of a template dir") { use_stdin = true }
```

Comment:
I agree with Danny that it isn't ideal to have an additional flag if we can help it. Could we use the unix convention of `-` representing stdin? Ultimately, it could be nice to make this arg feel more like kubectl's, which uses `--filename` for both files and directories, can combine multiple instances of the option into a list, and accepts `-` to mean stdin.

Reply:
That sounds reasonable. So you'd like to be able to use `--template-dir` to be used multiple times (or comma-separated values, I suppose). WDYT about the following 2 options:

1- Serially call kubernetes-deploy for each `template-dir`
2- When multiple template-dirs exist and/or stdin is used, place the contents of _all_ the template dirs + stdin in a temp directory and call `deploy_task` just once.

I'm leaning towards 2, myself.

Reply:
Option 2 👍 
Pruning would cause problems for option 1.

Reply:
>...files and directories, can combine multiple instances of the option into a list, and accepts `-` to mean stdin.

It turns out `ARGF` supports this out of the box 👍